infrastructure/main.tf

```diff
@@ -64,3 +64,14 @@ module "consumer" {
     FTP_PUBLIC_KEY            = "${replace(data.vault_generic_secret.ftp_public_key.data["value"], "\\n", "\n")}"
   }
 }
+
+module "key-vault" {
+  source              = "git@github.com:contino/moj-module-key-vault?ref=master"
+  product             = "${var.product}"
+  env                 = "${var.env}"
+  tenant_id           = "${var.tenant_id}"
+  object_id           = "${var.jenkins_AAD_objectId}"
+  resource_group_name = "${var.product}-producer-${var.env}"
+  # dcd_cc-dev group object ID
```

Comment:
could remove this comment ?

Reply:
I don't know - maybe it's better to know what group is related with that identifier?